src/progpy/prognostics_model.py

```diff
@@ -1411,15 +1426,29 @@ def optimization_fcn(params):
                     # If it doesn't work (i.e., throws an error), don't use it
             return err
         
-        params = np.array([self.parameters[key] for key in keys])
+        params = []
+        for key in keys:
+            if isinstance(key, (tuple, list)):
+                tmp = self.parameters
+                for key_element in key[:-1]:
+                    tmp = tmp[key_element]
+                params.append(tmp[key[-1]])
+            else:
+                params.append(self.parameters[key])
```

Comment:
It looks like this used to be a numpy array and now it's a list. I don't see it causing any issues but just want to confirm that it's okay that we changed this type.